_observing-your-data/alerting/monitors.md

```diff
@@ -138,7 +138,15 @@ POST _nodes/reload_secure_settings
 1. Specify a name for the monitor.
 1. Choose either **Per query monitor**, **Per bucket monitor**, **Per cluster metrics monitor**, or **Per document monitor**.
 
-Per query monitors run your specified query and then check whether the query's results trigger any alerts. Per bucket monitors let you select which fields to create buckets and categorize your results into those buckets. The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Each of those buckets can trigger an alert, but query-level monitors can only trigger one alert at a time.
+OpenSearch supports the following types of monitors:
+
+**Per query monitors** run your specified query and then check whether the query's results trigger any alerts. Query-level monitors can only trigger one alert at a time. 
+**Per bucket monitors** let you select which fields to create buckets and categorize your results into those buckets. The Alerting plugin runs each bucket's unique results against a script you define later, so you have finer control over which results should trigger alerts. Furthermore, each bucket can trigger an alert.
+
+The maximum number of monitors you can create is **1,000**. If you want to change the default maximum number of alerts for your cluster to below **1,000**, you can configure the maximum number of alerts by calling the cluster settings API `plugins.alerting.monitor.max_monitors`.
```

Comment:
@natebower Does the max number of monitors (1,000) count as a constraint? In that case, it shouldn't be bold, right?

Reply:
It should be bold if it appears as a UI element, if our formatting guidelines indicate that it should be, or if we particularly want to emphasize it. Otherwise, no.